visivo/commands/deploy_phase.py

```diff
@@ -1,19 +1,117 @@
 import click
 import requests
 import json
+import os
+import asyncio
+import aiofiles
+import httpx
+from time import time
+from tenacity import retry, stop_after_attempt, wait_fixed
 from visivo.commands.utils import get_profile_file, get_profile_token
 from visivo.discovery.discover import Discover
 from visivo.logging.logger import Logger
 from visivo.parsers.serializer import Serializer
 from visivo.parsers.parser_factory import ParserFactory
 
+# Limit concurrent uploads to avoid overloading the API
+semaphore = asyncio.Semaphore(50)
```

Comment:
I would be curious what the performance improvement is with a lower number, mostly I would be surprised if there is a large difference between 10 and 50

Reply:
@tgsoverly At ten it's actually much slower - 251.3

Reply:
I do wonder if we should be uploading more of the trace files in a single call rather than one call for every small file.

Reply:
Oh wow.